### DISCUSSION TITLE
fetch throttling

### DISCUSSION THREAD

**Discussion by alexanderGalushka** [denoland/deno#13505] [Q&A]
Is there a way to throttle fetch in deno?
wonder how is it solved in [deno deploy](https://deno.com/deploy)

**Comment by kitsonk** [comment]
What do you mean by fetch throttling? It could mean lots of things and requires clarification to provide a sensible response.

**Reply by alexanderGalushka** [reply]
limit the number of requests issued by fetch

**Reply by kitsonk** [reply]
What would be the use case for that?

There is no built in limits outside what the underlying OS handles. Deploy as a platform has anti-abuse and anti-TOS limits built into it, to preserve the availability of the platform, but nothing available to the end user.
